samples/lulesh/lulesh_sample1_pool_macosx.yaml

```diff
@@ -0,0 +1,90 @@
+description:
+    name: lulesh_sample1
+    description: A sample LULESH study that downloads, builds, and runs a parameter study of varying problem sizes and iterations.
+
+env:
+    variables:
+        # DEFAULTS FOR MONTECARLO PGEN EXAMPLE
+        SMIN:        10
+        SMAX:        30
+        TRIALS:      50
+        ITER:        100
+
+        OUTPUT_PATH: ./sample_output/lulesh
+
+    labels:
+        outfile: $(SIZE.label).$(ITERATIONS.label).log
+
+    dependencies:
+      git:
+        - name: LULESH
+          path: $(OUTPUT_PATH)
+          url: https://github.com/LLNL/LULESH.git
+          tag: 2.0.3
+
+batch:
+    shell: /bin/bash
+    type: local_pool
+    num_workers: 3
```

Comment:
we'll definitely need to spend some effort on whatever nomenclature is settled on for this stuff to avoid confusion with workers/cores/tasks/jobs/...

Reply:
Should I rename this to num_processes instead? I feel like that would be a better description that doesn't alias to other things.